scripts/check-includes.py

```diff
@@ -15,10 +15,10 @@
 def main(argv):
     argparser = ArgumentParser()
     argparser.add_argument('--generated-includes-dir', action='append',
-                           help='Directory where generated includes are located.')
-    argparser.add_argument('--file', type=open, help='File to check.')
+                           help='Directory where generated includes are located.',required=True)
```

Comment:
the linter complains about the missing space after `,`

Reply:
I have addressed all the comments and committed again can you please take a look thanks